src/Hardware_Reg_List.md

```diff
@@ -0,0 +1,124 @@
+$FF0F      | [IF]               | Interrupt flag                                                    | R/W                 | All
+$FF10      | [NR10]             | Channel 1 sweep                                                   | R/W                 | All
+$FF11      | [NR11]             | Channel 1 length timer & duty cycle                               | Mixed               | All
+$FF12      | [NR12]             | Channel 1 volume & envelope                                       | R/W                 | All
+$FF13      | [NR13]             | Channel 1 wavelength low                                          | W                   | All
+$FF14      | [NR14]             | Channel 1 wavelength high & control                               | Mixed               | All
+$FF16      | [NR21]             | Channel 2 length timer & duty cycle                               | Mixed               | All
+$FF17      | [NR22]             | Channel 2 volume & envelope                                       | R/W                 | All
+$FF18      | [NR23]             | Channel 2 wavelength low                                          | W                   | All
+$FF19      | [NR24]             | Channel 2 wavelength high & control                               | Mixed               | All
+$FF1A      | [NR30]             | Channel 3 DAC enable                                              | R/W                 | All
+$FF1B      | [NR31]             | Channel 3 length timer                                            | W                   | All
+$FF1C      | [NR32]             | Channel 3 output level                                            | R/W                 | All
+$FF1D      | [NR33]             | Channel 3 wavelength low                                          | W                   | All
+$FF1E      | [NR34]             | Channel 3 wavelength high & control                               | Mixed               | All
+$FF20      | [NR41]             | Channel 4 length timer                                            | W                   | All
+$FF21      | [NR42]             | Channel 4 volume & envelope                                       | R/W                 | All
+$FF22      | [NR43]             | Channel 4 frequency & randomness                                  | R/W                 | All
+$FF23      | [NR44]             | Channel 4 control                                                 | Mixed               | All
+$FF24      | [NR50]             | Master volume & VIN panning                                       | R/W                 | All
+$FF25      | [NR51]             | Sound panning                                                     | R/W                 | All
+$FF26      | [NR52]             | Sound on/off                                                      | Mixed               | All
+$FF30-FF3F | [Wave pattern ram] | Wave pattern RAM                                                  | R/W                 | All
```

Comment:
```suggestion
$FF30-FF3F | [Wave RAM] | Storage for one of the sound channels' waveform | R/W                 | All
```
This is merely one possibility, but I just find the term "wave pattern ram" very nondescript and opaque if you don't have a background in the domain.